src/Apps/Artist/ArtistApp.tsx

```diff
@@ -28,13 +23,7 @@ const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children }) => {
     <>
       <ArtistMetaFragmentContainer artist={artist} />
 
-      <AnalyticsContext.Provider
-        value={{
-          contextPageOwnerId: artist.internalID,
-          contextPageOwnerSlug,
-          contextPageOwnerType,
-        }}
-      >
+      <Analytics contextPageOwnerId={artist.internalID}>
```

Comment:
@dzucconi - Now that we're here doing this, maybe we could improve the api of this a bit even further... thoughts on 

```tsx
<Analytics internalID={artist.internalID}>
```

Nobody understands the contextPage redshift whatever idioms. if its always gonna be an `internalID`, lets just make that the prop. Ditto for `slug`

Reply:
I'm fine with that; we still want to pull them out though as `contextPageId` etc. It's confusing because in this case it means context in the normal sense of the word, not the React sense.

Reply:
Totally, just thinking that having `contextPageOwnerId` littered everywhere on init code (which actually just means `internalID` to the end user of the component) feels a bit better. What the user does with the data returned from the hook later is deliberate and intentional so yah, however that should be is fine

Reply:
(ignore me!)